Cargo.toml

```diff
@@ -11,19 +11,19 @@ repository = "https://github.com/stackabletech/hdfs-operator"
 
 [workspace.dependencies]
 anyhow = "1.0"
-built = { version = "0.6", features = ["chrono", "git2"] }
+built = { version = "0.7.2", features = ["chrono", "git2"] }
```

Comment:
This was actually intentional, because I thought that I remembered that 0.7.0 doesn't work for us, as this still uses an old git2 version which has a critical vulnerability that we want to get rid of. But actually looking at the code of built it seems that this was indeed bumped in 0.7.0 already, so is all good.